include/CppUTestExt/MockNamedValue.h

```diff
@@ -60,13 +60,41 @@ class MockFunctionComparator : public MockNamedValueComparator
 };
 
 /*
+ * MockNamedValueCopier is an interface that needs to be used when creating Copiers.
+ * This is needed when copying values of non-native type.
+ */
+
+class MockNamedValueCopier
+{
+public:
+    MockNamedValueCopier() {}
+    virtual ~MockNamedValueCopier() {}
+
+    virtual void copy(void* dst, const void* src)=0;
```

Comment:
You are right. The original suggestion by @random85 had `const void*`, but that was silently cast away by casting to (Object*) for dst. `void *` it shall be.